plugins/out_stackdriver/fluent-bit.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Looks like you added a config file for your editor by accident?

Reply:
Yeah, that is what I get for doing that push late-night. Fixing.